charts/argocd-workflows-apps/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: argocd-workflows-apps
+description: An argocd app to deploy apps in the namespace
```

Comment:
This description could probably be more descriptive - since there are only two apps we can probably enumerate them